modules/cloud-credential-operator.adoc

```diff
@@ -8,8 +8,290 @@
+section of the xref:../installing/installing_azure/installing-azure-account.adoc[Configuring an Azure account]
+page.
+
+|GCP
+|* `resourcemanager.projects.get`
+* `serviceusage.services.list`
+* `iam.serviceAccountKeys.create`
+* `iam.serviceAccountKeys.delete`
+* `iam.serviceAccounts.create`
+* `iam.serviceAccounts.delete`
+* `iam.serviceAccounts.get`
+* `iam.roles.get`
+* `resourcemanager.projects.getIamPolicy`
+* `resourcemanager.projects.setIamPolicy`
+
+|====
+
+[discrete]
+===== Mint mode with removal or rotation of the admin-level credential
+Mint mode with removal or rotation of the admin-level credential is supported
+for AWS in {product-title} version 4.4 and later.
+
+This option requires the presence of the admin-level credential during
+installation, but the credential is not stored in the cluster permanently and
+does not need to be long-lived.
+
+After installing {product-title} in mint mode, you can remove the admin-level
+credential Secret from the cluster. If you remove the Secret, the CCO requests a
+read-only credential that allows it to verify whether all `credentialsRequest`
+CRs have their required permissions. Once removed, the associated credential can
+be destroyed on the underlying cloud if desired.
+
+The admin-level credential is not required unless something that requires an
+admin-level credential needs to be changed, for instance during an upgrade.
+Prior to each upgrade, you must reinstate the credential Secret with the
+admin-level credential. If the credential is not present, the upgrade might be
+blocked.
+
+[discrete]
+==== Passthrough mode
+Passthrough mode is supported for AWS, Azure, GCP, {rh-openstack-first},
+{rh-virtualization-first}, VMware vSphere.
+
+In passthrough mode, the CCO uses a provided non-admin cloud credential to run
+the installer. The credential has permissions to perform the installation and
+complete the operations that are required by components in the cluster, but
+cannot create additional credentials.
+
+[discrete]
+===== Passthrough mode permissions requirements
+The credential you provide for passthrough mode in AWS, Azure, or GCP must have
+all the requested permissions for all `credentialsRequest` CRs that are required
+by the version of {product-title} you are running or installing. To locate the
+`credentialsRequest` CRs that are required for your cloud provider see
+_Manually creating IAM_ for xref:../installing/installing_aws/manually-creating-iam.adoc[AWS],
+xref:../installing/installing_azure/manually-creating-iam-azure.adoc[Azure], or
+xref:../installing/installing_gcp/manually-creating-iam-gcp.adoc[GCP].
+
+To install an {product-title} cluster on {rh-openstack-first}, the CCO requires a
+credential with the following privileges:
+
+//6 Oct 2020: This table is inaccurate and will be revised with new info as soon as available.
+.Required {rh-openstack-first} privileges
```

Comment:
Note: waiting on corrected data here but don't want to hold the review for it. Will update ASAP.